File: NthTerm.c
#include <stdio.h>

int main() {
    int a, d, n, nth;

    printf("Enter first term: ");
    scanf("%d", &a);

    printf("Enter common difference: ");
    scanf("%d", &d);

    printf("Enter term number (n): ");
    scanf("%d", &n);

    nth = a + (n - 1) * d;

    printf("The %dth term of the AP is: %d", n, nth);

    return 0;
}

File: NthTerm.java
import java.util.Scanner;

public class APNthTerm {
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        System.out.print("Enter first term: ");
        int a = sc.nextInt();

        System.out.print("Enter common difference: ");
        int d = sc.nextInt();

        System.out.print("Enter term number (n): ");
        int n = sc.nextInt();

        int nth = a + (n - 1) * d;

        System.out.println("The " + n + "th term of the AP is: " + nth);

        sc.close();
    }
}
